Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -34,6 +35,170 @@
 // happen with the Freetype versions
 static const bool NO_FREETYPE = true;
 
+namespace {
```

Comment:
One of the really nice things about the catch tests is how easy it is to add a new test: you just add the test in one place and it gets run.
With the way you've set this up authors of new tests would have to first add their test and then add the hash for the test file up here. It's not that much extra work, but it's unnecessary.
You could instead have `check_file_hash()` take the target hash value as an optional argument which defaults to empty. When it's pretsent it gets used and when not it's ignored.
I think this makes things easier and clearer.

There's an argument that the same thing could be done for the old-style test files, but I don't feel quite as strongly about that

Reply:
I set it up like that originally in test1.cpp but every test needed a Freetype and non-Freetype version so was 5 lines with the #ifdef...#else...#endif and I thought it was tidier (and a lot less tedious for all the historical cases) to put it all in check_file_hash().
That's not an issue here, I guess, as we're explicitly not using FT.  I suppose different behaviour for the same function name is ok in the test code.

Reply:
good point about the different test versions.
as I said, I feel most strongly about keeping the catch tests easy to write, so I'm fine with just having the change made there

Reply:
And the other thing is, if we ever have to bulk-change the hash codes for some reason, it's a lot easier to make a new map initialisation block by piping output through awk.  It's a lot of tedious cut-and-paste to put it in at every call to check_file_hash().  As it stands, if there isn't a hash code in the map for a file, it just leaves the file as is.

Reply:
How about leaving the map as is, but giving the option for passing the expected value through as an alternative?